app/helpers/course/discussion/topics_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Course::Discussion::TopicsHelper
+  # Display code lines in file.
+  #
+  # @param [Course::Assessment::Answer::ProgrammingFile] file The code file.
+  # @param [Integer] line_start The one based start line number.
+  # @param [Integer] line_end The one based end line line number.
+  # @return [String] A HTML fragment containing the code lines.
+  def display_code_lines(file, line_start, line_end)
```

Comment:
@lowjoel A helper method is added here to display the code lines in the annotation.